### ISSUE TITLE
`use Type::Params ':all'` does actually work

### ISSUE THREAD

**Issue by tobyink** [Ovid/moosex-extended#56]
https://github.com/Ovid/moosex-extended/blob/b39b28389af7dc8d4d097dd85f1264b0291966fa/lib/MooseX/Extended/Types.pm#L10

Exporter::Tiny implicitly defines `:all` and `:default` tags:

```
$ perl -MType::Params=:all -E'say \&compile_named_oo'
CODE(0x557d0bd77e18)
```

**Comment by Ovid** [comment]
Ah! I was looking in your type code for `%EXPORT_TAGS` and since I didn't see it, I assumed it wasn't there. This will be fixed in the next release. Thank you :)

**Comment by Ovid** [comment]
This is fixed on version 0.33 on its way to the CPAN now.
